src/components/ProBanner.tsx

```diff
@@ -0,0 +1,20 @@
+import { useGlobalContext } from "../context/Global";
+
+const ProBanner = () => {
+    const { t } = useGlobalContext();
+
+    return (
+        <div class="banner banner-yellow">
+            <a 
+                href="https://docs.boltz.exchange/boltz-client/boltz-pro" 
+                target="_blank" 
+                rel="noopener noreferrer" 
+                style="text-decoration: underline; color: inherit; text-transform: uppercase;"
+            >
+                {t("pro_banner")}
+            </a>
```

Comment:
I guess we should add the "external link" icon here:
```suggestion
            <a
                href="https://docs.boltz.exchange/boltz-client/boltz-pro"
                target="_blank"
                rel="noopener noreferrer">
                {t("pro_banner")}
                <OcLinkexternal2 size={18} />
            </a>
```
And on `nav.scss`:
```
.banner-yellow {
    background: vars.$primary;
    color: #14283f;

    a {
        display: inline-flex;
        align-items: center;
        gap: 0.5rem;

        text-decoration: underline;
        text-transform: uppercase;
        color: #14283f;
    }

    a:hover {
        color: currentColor;
    }
}
```
![image](https://github.com/user-attachments/assets/f44cfe2b-2ae7-4ad8-a62b-d547e714f6cc)

Reply:
Agreed